boa_interner/src/lib.rs

```diff
@@ -67,44 +67,161 @@
     clippy::missing_errors_doc,
     clippy::as_conversions,
     clippy::let_unit_value,
-    rustdoc::missing_doc_code_examples
+    // TODO deny once false positive are fixed (https://github.com/rust-lang/rust-clippy/issues/9076).
+    clippy::trait_duplication_in_bounds,
+    rustdoc::missing_doc_code_examples,
```

Comment:
We might want to add examples in the documentation, right? Or maybe create an issue to track this.

Reply:
I'll create an issue, since this crate didn't have examples before this PR.